src/main/java/core/basesyntax/GetArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface GetArea {

```

Comment:
The interface name `GetArea` is not following the Java naming conventions. Interfaces should be adjectives. In this case, `AreaCalculable` would be a more appropriate name. Refer to the checklist: 'Don't use verbs for class/interface names'.